library/Businessprocess/Renderer/TileRenderer/NodeTile.php

```diff
@@ -102,11 +103,7 @@ public function render()
 
             $this->add($link);
         } else {
-            $this->add(Html::tag(
-                'span',
-                ['class' => 'missing-node-msg'],
-                sprintf('Trying to access a missing business process node "%s"', $node->getNodeName())
-            ));
+            $this->add(new Link($node->getAlias(), (string) $this->getMainNodeUrl($node)));
```

Comment:
Don't cast here. By casting you'll get an encoded url back for use directly in HTML. Use `->getAbsoluteUrl()` instead, this way you don't have a problem with an encoded url being parsed back into a `ipl\Web\Url`.